docs/metadata.md

```diff
@@ -0,0 +1,49 @@
+Adding business context to a system using metadata
+==================================================
+
+Adding business context to Sigrid makes it easier to interpret the results. For example, a below-market average maintainability rating of 2 stars might seem like a problem, but this depends entirely on the context. If the system is no longer actively maintained and will be decommissioned by the end of the year, such a rating might be perfectly acceptable. But if that system is new, uses a modern technology, and is business critical, such a rating would be considered a red flag. In both cases the technical conclusion is identical, it's the context that determines the urgency.
+
+This context information is called *metadata* in Sigrid. Adding metadata can be done in 3 different ways:
+
+### Option 1: Adding metadata in Sigrid
+
+This is by far the simplest approach: in Sigrid, simply go to the system settings option in the menu and select the "metadata" option. 
+
+<img src="images/metadata-ui.png" width="600" />
```

Comment:
Should we indicate that the screenshot displays only a part of the UI? Most metadata fields are not shown. In particular, the lifecycle field is not shown, which would connect the screenshot to the motivation for recording metadata in the first paragraph.

Reply:
I'll indeed add that just to be sure.